app/web/src/components/layout/navbar/WorkspaceSettingsMenu.vue

```diff
@@ -29,20 +35,43 @@
 <script setup lang="ts">
 import { DropdownMenuItem, Icon } from "@si/vue-lib/design-system";
 import { ref } from "vue";
+import { useRouter, useRoute } from "vue-router";
 import WorkspaceImportModal from "@/components/WorkspaceImportModal.vue";
 import WorkspaceExportModal from "@/components/WorkspaceExportModal.vue";
 import { useWorkspacesStore } from "@/store/workspaces.store";
+import { useFeatureFlagsStore } from "@/store/feature_flags.store";
+import { useChangeSetsStore } from "@/store/change_sets.store";
 import NavbarButton from "./NavbarButton.vue";
 
 const AUTH_PORTAL_URL = import.meta.env.VITE_AUTH_PORTAL_URL;
 const importModalRef = ref<InstanceType<typeof WorkspaceImportModal>>();
 const exportModalRef = ref<InstanceType<typeof WorkspaceExportModal>>();
 
 const workspacesStore = useWorkspacesStore();
+const featureFlagsStore = useFeatureFlagsStore();
+const changeSetStore = useChangeSetsStore();
+const router = useRouter();
+const route = useRoute();
 
 const openWorkspaceDetailsHandler = () => {
   const currentWorkspace = workspacesStore.urlSelectedWorkspaceId;
   if (!currentWorkspace) return;
   window.open(`${AUTH_PORTAL_URL}/workspace/${currentWorkspace}`, "_blank");
 };
+
+const showViz = () => {
+  if (!changeSetStore.selectedChangeSetId) return false;
+  if (!featureFlagsStore.FEAT_GRAPHVIZ) return false;
+  return true;
+};
```

Comment:
If this is not created as a `computed` field it will not have reactivity, so it would not respond to a selected changeset change, for example 

You could do something like:
```suggestion
const showViz = computed(() => {
  if (!changeSetStore.selectedChangeSetId) return false;
  if (!featureFlagsStore.FEAT_GRAPHVIZ) return false;
  return true;
});
```
and just use `showViz` instead of calling the function when refering to it

Reply:
Good catch